roles/include_components/tasks/main.yml

```diff
@@ -17,11 +17,17 @@
 
 - name: Track RPMs
   ansible.builtin.include_tasks: track_rpm.yml
-  loop: "{{ ic_rpms }}"
+  loop: "{{ ic_rpms | default([]) }}"
   when:
     - item is not in
       ( (ic_gits + ic_dev_gits) |
         map('regex_replace', '^(?:.*/(.+)|.*/([^/]+))$', '\\1') |
         map('regex_replace', 'python', 'python3') | list)
     - item in ansible_facts.packages
+
+- name: Track Github components
```

Comment:
I'm a bit concerned about the naming here. If the variable is `ic_repo_commits` and it's talking about code commits, isn't it better to keep that naming instead of using Github? Something like `Track code commit components`

Reply:
The code only works for GH see https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/113#discussion_r1445124954

Reply:
@ramperher let me rename loop vars to `ic_githubs` to highlight that it'll only work for GH